### PR TITLE
GEODE-4737 Document JSON spec for gfsh command options

### PR DESCRIPTION
**Pull request by karensmolermiller**

### CODE REVIEW

File: geode-docs/tools_modules/gfsh/json_in_gfsh.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Some `gfsh` commands allow JSON specifications within some options.
+These JSON strings specify pairs of strings.
+Each pair specifies one initialization property.
+
```
Comment (sboorlagadda) [inline geode-docs/tools_modules/gfsh/json_in_gfsh.html.md.erb]: I would rephrase this to say - "These JSON strings specify initialization properties in form of key/value pairs" or similar.

Comment (pdxrunner) [reply geode-docs/tools_modules/gfsh/json_in_gfsh.html.md.erb]: +1

**Comment by sboorlagadda** [comment]
👍
